code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -33,9 +36,10 @@
 		var/decl/reagent/R = decls_repository.get_decl(T)
 		reagent_names += R.name
 
+	update_icon()
 	START_PROCESSING(SSprocessing, src)
 
-/obj/item/reagent_containers/hypospray/borghypo/update_icon()
+/obj/item/reagent_containers/hypospray/borghypo/update_icon() // To do: Will work on the Medical Synthesizer in a future PR -Vrow
```

Comment:
We should have your TODO in the repo right? Just make yourself a textfile or something to track your TODO list.

Reply:
There are random To do's scattered around the code. This one I put in since I do have some stuff ready for the medical synthesizer, which will be worked more actively once this one gets merged so I don't have merge conflicts between the two.